base_address_parish/i18n/base_address_parish.po

```diff
@@ -0,0 +1,83 @@
+# Translation of Odoo Server.
```

Comment:
You should generate this file like .pot file, then oca bot generate per project Webtable space for transalate

https://translation.odoo-community.org/